Header header
uint16 V1
uint16 V2
uint16 V3
uint16 V4
uint16 C1
uint16 C2
uint16 C3
uint16 C4
uint16 THRUST1
uint16 STRAIN1
uint16 THRUST2
uint16 STRAIN2
uint16 THRUST3
uint16 STRAIN3
uint16 THRUST4
uint16 STRAIN4
uint16 RPM1
uint16 RPM2
uint16 RPM3
uint16 RPM4
